res/database/Default/database_news.xml

```diff
@@ -1206,56 +1407,60 @@
+		<news guid="2739e75a-51fb-400e-a26a-5fce3543a108" type="0" thread_guid="2e9f56cb-f123-4626-8951-7e37fd666ad7" creator="8936" created_by="jorgaeff" comment="https://en.wikipedia.org/wiki/Mathias_Rust | https://en.wikipedia.org/wiki/Cessna_172">
 			<title>
 				<de>Pilot landet auf Rotem Platz in Moskau</de>
 				<en>Pilot lands on Red Square in Moscow</en>
 				<pl>Pilot ląduje na Placu Czerwonym w Moskwie</pl>
 			</title>
 			<description>
-				<de>Der deutsche Pilot Markus Trust ist mit seiner Schessnah auf dem Roten Platz in Moskau gelandet.</de>
-				<en>The German pilot Markus Trust has landed with his Schessnah on the Red Square in Moscow.</en>
-				<pl>Niemiecki pilot Markus Trust wylądował swoim Schessnahem na Placu Czerwonym w Moskwie.</pl>
+				<de>Der deutsche Pilot ${mathias} ${rust} ist mit seiner Schessnah auf dem Roten Platz in Moskau gelandet.</de>
+				<en>The German pilot ${mathias} ${rust} has landed with his Schessnah on the Red Square in Moscow.</en>
+				<pl>Niemiecki pilot ${mathias} ${rust} wylądował swoim Schessnahem na Placu Czerwonym w Moskwie.</pl>
 			</description>
-			<data genre="4" price="0.8" quality="80" fictional="0" flags="2" happen_time="5,1987,1987,5,5,10,30" /> <!-- actual date: 1987-05-28, leaving wiggle room for random choice and deactivation of duplicate -->
+			<variables>
+				<mathias>${.person:"95bb70af-624f-4396-a09d-698d284a59f3":"firstname"}</mathias>
+				<rust>${.person:"95bb70af-624f-4396-a09d-698d284a59f3":"lastname"}</rust>
+			</variables>
+			<data genre="4" price="0.8" quality="80" fictional="0" flags="2" happen_time="5,1987,1987,4,6,1,30" /> <!-- actual date: 1987-05-28, also leaving wiggle room for random choice and deactivation of duplicate -->
 			<effects>
 				<!-- "in 1 Tagen 10-16 Uhr" -->
 				<effect trigger="happen" type="triggernews" time="1,2,3" news="ab5b5b9a-1b67-4fd7-80a9-e6eca6b705e6" />
 				<effect trigger="happen" type="modifyNewsAvailability" enable="0" news="d58a07ac-8a3f-4dd0-b2db-6e0b742efd23" />
 			</effects>
 		</news>
 
-		<news guid="0e11a4da-1982-4d81-bd18-713201c7af6f" type="2" thread_guid="f4658e9a-d45f-4919-8547-0294d9e09552" creator="8936" created_by="jorgaeff" comment="TODO: ref persons Michailow, Markus Trust">
+		<news guid="0e11a4da-1982-4d81-bd18-713201c7af6f" type="2" thread_guid="7ce7e209-724e-43dc-bdb6-3fd76fb90cde" creator="8936" created_by="jorgaeff" comment="TODO: can't use variables here because thread has no start entry (may be called be two other threads); but maybe they can all use the same thread_id">
 			<title>
-				<de>Michailow entlässt Minister</de>
-				<en>Michailow dismisses minister</en>
-				<pl>Michailow odwołuje ministra</pl>
+				<de>${gorbachev} entlässt Minister</de>
```

Comment:
This news item is triggered by an item that does not define mikhail or gorbachev.

Reply:
it is now, i have "merged" the thread (though it's one thread by two different contributors now).
(though i forget to replace the guid references in other instances.)

Reply:
Now after you changed something? When I wrote the comment. my test creation of news chains showed unresolved variables. So at least in one of the possible chains, the variables were not defined.

Reply:
Ok it didn't seem necessary but I've added to the other thread as well now.

Reply:
For the variables to be resolvable it does not suffice that there is some potential path where it is defined. Every path leading to a news item that uses a variable must also contain its definition.